profile_library/xiaomi/philips.light.hbulb/model.json

```
@@ -0,0 +1,17 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2024-12-08T13:31:31.689330",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Shelly Plus Plug S",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.16.2:docker"
+  },
+  "name": "Philips Smart LED Bulb",
+  "standby_power": 0.9,
+  "aliases": [
+    "philips.light.hbulb"
+  ]
```

```suggestion
  "standby_power": 0.9
```

alias can be removed as the directory name (model id) is already the same.
Powercalc looks at directory first, and additionally any of the defined aliases.